recruiter/templates/recruiter/company_created_successfully.html

```diff
@@ -0,0 +1,6 @@
+{% extends 'base.html' %} {% block content %}
+<h3>
+  You have successfully created new Company!
+  <a href="/login">let's login</a>
```

Comment:
I think you mixed up the message here with the message on `recruiter/templates/recruiter/recruiter_created_successfully.html`
The message here should say "go back and sign up as a Recruiter"
and the message on the other page should say "lets login"

Reply:
done